index.html

```diff
@@ -197,6 +197,7 @@ <h3>Name Computation</h3>
   <section id="mapping_additional_nd_description">
     <h3>Description Computation</h3>
     <p>If <a class="property-reference" href="#aria-describedby"><code>aria-describedby</code></a> is present, <a class="termref">user agents</a> MUST compute the accessible description by concatenating the text alternatives for elements referenced by an <code>aria-describedby</code> attribute on the current element. The text alternatives for the referenced elements are computed using a number of methods, outlined below in the section titled <a href="#mapping_additional_nd_te">Accessible Name and Description Computation</a>. </p>
+    <p>However, if <code>aria-describedby</code> is NOT present, and <a class="property-reference" href="#aria-description"><code>aria-description</code></a> IS present, <a class="termref">user agents</a> MUST use the attribute value of <code>aria-description</code> as the accessible description.
```

Comment:
Nit: I wouldn't capitalize "NOT" or "IS."

Reply:
Done